includes/class-sensei-modules.php

```diff
@@ -1268,8 +1268,8 @@ public function get_course_modules($course_id = 0) {
      */
     public function enqueue_styles() {
 
-        wp_register_style($this->taxonomy . '-frontend', esc_url($this->assets_url) . 'css/modules-frontend.css', Sensei()->version );
-        wp_enqueue_style($this->taxonomy . '-frontend');
+        // wp_register_style($this->taxonomy . '-frontend', esc_url($this->assets_url) . 'css/modules-frontend.css', Sensei()->version );
```

Comment:
Since you're commenting out all the styles that were enqueued before, you could get rid of the function altogether.